### PR TITLE
Update font.cpp

### PR DESCRIPTION
**Pull request by AlexGlide**

### CODE REVIEW

**Comment by AlexGlide** [comment]
Hmm, what the heck was? Am I need commit file again?

File: test152/font.cpp [hunk 1]
```diff
@@ -777,38 +777,4 @@ const unsigned char pic_HARRIS[]= {
     0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
     0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
 };
-const unsigned char pic_XVF[] = {
```
Comment (tarxvftech) [inline test152/font.cpp:780]: This is the new pic_XVF splash image, displayed on boot. You might be able to re-add it in another commit to this branch, and if it worked then the CI pipeline should succeed.

Comment (AlexGlide) [reply test152/font.cpp:780]: Maybe for first time we can block this part of code? And later we will be able to sort out the problem?

Comment (tarxvftech) [reply test152/font.cpp:780]: I think I can edit this pull request, standby

Comment (tarxvftech) [reply test152/font.cpp:780]: Apparently yes! Fixed!

**Pull request merged**